sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,20 @@
+DROP TABLE IF EXISTS `achievement_reward_locale`;
+CREATE TABLE `achievement_reward_locale` (
+	`ID` INT(10) UNSIGNED NOT NULL DEFAULT '0',
+	`Locale` VARCHAR(4) NOT NULL,
+	`Subject` TEXT NULL,
+	`Text` TEXT NULL,
+	PRIMARY KEY (`ID`, `Locale`)
+);
+
+-- The old table is empty on default, but migrate data to new table just in case
+INSERT INTO `achievement_reward_locale` (`ID`, `Locale`, `Subject`, `Text`) SELECT `entry`, "koKR", `subject_loc1`, `text_loc1` FROM `locales_achievement_reward` WHERE `text_loc1` != "";
+INSERT INTO `achievement_reward_locale` (`ID`, `Locale`, `Subject`, `Text`) SELECT `entry`, "frFR", `subject_loc2`, `text_loc2` FROM `locales_achievement_reward` WHERE `text_loc2` != "";
+INSERT INTO `achievement_reward_locale` (`ID`, `Locale`, `Subject`, `Text`) SELECT `entry`, "deDE", `subject_loc3`, `text_loc3` FROM `locales_achievement_reward` WHERE `text_loc3` != "";
```

Comment:
Shouldn't there be a `VALUES` keyword before the keyword `SELECT` in these queries, possibly followed by parentheses for the `SELECT` statements?

Reply:
Can't really say, I'm not the big SQL guy. 
But this code works and migrates all data from the old table to the new one, so I'm fine with it.

Reply:
@illfated You use values if you define values yourself
`INSERT INTO (a, b, c) VALUES 'a', b, 'c'`
However, this INSERT INTO basically inserts values you select FROM a different table instead of typing them yourself.

That's the idea anyway. Instead of defining the values yourself, you simply take them from a different table.